strict.js

```diff
@@ -81,7 +81,14 @@ module.exports = {
 		'@stylistic/js/linebreak-style': ['warn', 'unix'],
 		'@stylistic/js/operator-linebreak': ['warn', 'after'],
 		'@stylistic/js/space-before-blocks': ['warn', 'always'],
-		'@stylistic/js/space-before-function-paren': ['warn', 'always'],
+		'@stylistic/js/space-before-function-paren': [
+			'warn',
+			{
+				anonymous: 'always',
+				named: 'never',
```

Comment:
I think this change will make huge lint warnings from our repos which uses enact lint.
If I use this branch and do `npm run lint` from sandstone for example, I get tons of warnings like
```
...
/sandstone/WizardPanels/Panel.js
  15:19  warning  Unexpected space before function parentheses  @stylistic/js/space-before-function-paren

/sandstone/internal/$L/$L.js
  14:22  warning  Unexpected space before function parentheses  @stylistic/js/space-before-function-paren
  25:25  warning  Unexpected space before function parentheses  @stylistic/js/space-before-function-paren
  47:24  warning  Unexpected space before function parentheses  @stylistic/js/space-before-function-paren
...
```
I don't think this is good idea to solve the problem.